website/docs/reference/warehouse-setups/postgres-setup.md

```diff
@@ -15,6 +15,10 @@ meta:
   config_page: 'postgres-configs'
 ---
 
+:::info
```

Comment:
@mirnawong1 We can ship and then follow up on this, but I think these callouts are a great use case for reusables! It would keep the wording consistent and if we need to add any other context, we can do it in one place.

Reply:
sounds good Leona - i'm going to use the snippet and then merge this since you've approved it already (content-wise) i hope that's ok and gets this up for Katherine and our users.